### PR TITLE
[refactor]: astro-reactive-form types

### PR DESCRIPTION
**Pull request by amit-ksh**
What is this PR about? refactoring
- Refactoring the astro-reactive-form directory
- Maintaining the types of the package

Issue: #24  

Description of changes:
- created a separate types.ts file inside the astro-reactive-form directory storing all the types

Tasks:
- [x] I have ran the tests to make sure nothing is broken (see CONTRIBUTING.md)
- [x] I have mentioned the issue this PR is addressing

### CODE REVIEW

File: packages/astro-reactive-form/core/form-control.ts [hunk 1]
```diff
@@ -2,101 +2,55 @@
...
+	 }
+ 
+	 setValue(value: string) {
+		 this._value = value;
+	 }
+ 
+ }
+ 
+ /**
+	* TODO: Create classes for each control type
```
Comment (ayoayco) [inline packages/astro-reactive-form/core/form-control.ts:52]: I think we can clean up the empty lines and comments here as well :)

**Review by ayoayco — COMMENTED** [review]
Great clean up! I just have one suggestion :)

File: packages/astro-reactive-form/core/form-control.ts (hunk 1 above)
Comment (amit-ksh) [reply packages/astro-reactive-form/core/form-control.ts]: Done!

**Review by ayoayco — APPROVED** [review]
Already clearer with a simple clean up. Great update! 🚀

**Pull request merged**